db/seeds.rb

```diff
@@ -1,21 +1,68 @@
-User.destroy_all
-PathChallenge.destroy_all
 Pathway.destroy_all
 Challenge.destroy_all
-
+UserChallenge.destroy_all
+User.destroy_all
 puts "RUNNING SEED FILE"
 puts "-----------------"
-user = User.create!(first_name: "Ben", last_name: "Simpson", email: "ben@ben.com", password: "Testpassword1!", role: :mentor)
-puts "One user created"
 
-pathway = Pathway.create!(title: "Test 1", details: "This is a test pathway", difficulty: "beginner", user_id: user.id)
-puts "One pathway created"
+@mentor = User.create!(
+    first_name: "Mentor", last_name: "Test", email: "mentor@test.com", password: "Testpassword1!", role: "mentor")
+
+@mentee = User.create!(
+    first_name: "Mentee", last_name: "Test", email: "mentee@test.com", password: "Testpassword1!", role: "mentee")
+
+level = ["beginner", "intermidiate", "professional", "expert" ]
+
+mark = ["1", "2", "3", "4", "5" ]
 
-challenge = Challenge.create!(title: "Database number 1", details: "This is a question that needs answers regarding the database", user_id: User.last.id)
+def CreateChallenge(order)
+    Challenge.create!(
+        title: "Q #{order} Public Cloud and Private Cloud provider",
+        details: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
+        user_id: @mentor.id 
+    )
+
+    PathChallenge.create!(
+        challenge_id: Challenge.last.id,
+        pathway_id: Pathway.last.id, 
+        order: order  
+    )
+end
+
+5.times do |i| 
+    level_choice = rand(0..3) 
+    Pathway.create!( user_id: @mentor.id,
+    title: "Pathway, #{i}!",
```

Comment:
`    title: "Pathway, #{i + 1}!", `
So that pathways are named but not indexed at 0. I.e 
Pathway 1, Pathway 2...
Instead of Pathway 0, Pathway 1....